### PR TITLE
Add python support

### PR DESCRIPTION
**Pull request by youzhonghui**
Look like this:

a

### CODE REVIEW

**Comment by simurai** [comment]
Thanks. Might this PR would be good to add on top of https://github.com/atom/solarized-light-syntax/pull/16. I'll take another look at it.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "solarized-light-syntax",
   "theme": "syntax",
-  "version": "0.23.0",
+  "version": "0.23.1",
```
Comment (simurai) [inline package.json]: The version doesn't need to be changed in this PR. It will get updated automatically when publishing a new release of the theme.

**Comment by youzhonghui** [comment]
OK. Already fix the version.

**Comment by simurai** [comment]
Did you mean to close this PR? I guess #16 already came with Python support from Solarized dark.

**Comment by youzhonghui** [comment]
I noticed that the file struct changed. I will open an new PR later.

**Comment by simurai** [comment]
I see. Yeah, Solarized dark uses this: https://github.com/atom/solarized-light-syntax/blob/master/styles/languages/python.less

Probably a bit different than your version.

**Pull request closed without merging**